app/src/main/java/net/gsantner/markor/activity/DocumentActivity.java

```diff
@@ -271,14 +278,10 @@ public void setDocumentTitle(final String title) {
         }
     }
 
-    public void showTextEditor(@Nullable Document document, @Nullable File file, boolean fileIsFolder) {
-        showTextEditor(document, file, fileIsFolder, false);
-    }
-
-    public void showTextEditor(@Nullable Document document, @Nullable File file, boolean fileIsFolder, boolean preview) {
-
+    public void showTextEditor(@Nullable Document document, @Nullable File file, boolean fileIsFolder, boolean preview, final int... arg0fileLineNumber) {
```

Comment:
Same observation about using an overload. Or just use `Integer lineNumber` and pass in null wherever needed

Reply:
@adelobosko missed this one? :)